src/parsers/packets/CarStatusDataParser.ts

```diff
@@ -13,7 +13,8 @@ export class CarStatusDataParser extends F1Parser {
         .floatle('m_fuelInTank')
         .floatle('m_fuelCapacity');
 
-    if (packetFormat === 2019 || packetFormat === 2020) {
+    if (packetFormat === 2019 || packetFormat === 2020 ||
+        packetFormat === 2021) {
```

Comment:
In javascript you can do multiple shorthand conditional checks by doing `packetFormat === ( 2019 || 2020 || 2021) `
Same functionality, less code